matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/model/DeviceListResponse.kt

```diff
@@ -13,15 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.matrix.android.sdk.internal.session.sync.model
+
+package org.matrix.android.sdk.api.session.sync.model
```

Comment:
I understand this change, but I'm not a big fan of it... How we will decide in the future if some network data classes should be internal or not?

Reply:
As we want the network layer to be exposed, I guess every data classes will be public

Reply:
Otherwise, maybe we can just let the Flow encapsulates JsonDict instead of SyncResponse?

Reply:
IDK... Let's keep it like that for the moment.